src/ProgramBlockEditor.js

```diff
@@ -25,6 +26,9 @@ type ProgramBlockEditorProps = {
 };
 
 class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
+    commandBlock: Button
```

Comment:
Please add a constructor to initialize commandBlock:

    constructor(props: ProgramBlockEditorProps) {
        super(props);
        this.commandBlock = null;
    }